### PR TITLE
Implement responsive grid in blog page

### LINKED ISSUE #3475: Responsive Grid in Blog Page
**Actual Behaviour**
The blog page has large posts that take over the whole screen in laptops or desktops. The page has a very large image of blog and covers the whole screen

**Expected Behaviour**
The blog should have a responsive grid system. A nice looking blog page in mobile devices as well as on big screens

**Steps to reproduce it**
Go to `susi.ai/blog` and you will see the inappropriate interface 

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by MuhammadWasif**
Fixes #3475 
Changes: [Add here what changes were made in this pull request and if possible provide links showcasing the changes.]
Made a responsive grid with CSS in `/blog` page.
### Demo Link: 
https://awesome-poitras-b9f30d.netlify.com/
### Screenshots of the change:
sus_ai_grid_demo

### CODE REVIEW

**Comment by elit-altum** [comment]
This looks great! Maybe you could make the individual cards of equivalent (and maybe fixed) heights. The difference looks pronounced in smaller screens.

image

File: src/components/About/Blog/BlogItem.js [hunk 1]
```diff
@@ -142,6 +142,10 @@ const Title = styled.div`
 
 const Card = styled(_Card)`
   position: relative;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```
Comment (mohitm15) [inline src/components/About/Blog/BlogItem.js]: Use @media queries to fix the height difference in mobile view as said by @elit-altum

**Comment by snitin315** [comment]
Make it 3-4 cards per row.

**Comment by MuhammadWasif** [comment]
I think this can only be implemented if each blog has its own page. Then the cards can be arranged in a grid.

**Comment by mohitm15** [comment]
Is this issue is still open?

**Comment by mohitm15** [comment]
I like to work on this?? @akshatnitd

**Comment by snitin315** [comment]
I'm already working on this. @mohitm15

**Comment by mohitm15** [comment]
> I'm already working on this. @mohitm15

Have you claimed it before?

**Comment by snitin315** [comment]
Yes , checkout the corresponding issue.

**Comment by mohitm15** [comment]
Ok no problem

**Pull request closed without merging**